proof-generator/USAGE

```diff
@@ -0,0 +1,20 @@
+Usage
+
+  proof-generator --gpg-key <string> --keystore <file> --rpc-url <url> --output <file> [<option>..] 
+
+Options
+
+    --gpg-key <string>           GPG long key 
```

Comment:
I thought the key fingerprint was sufficient? Or do we only use it for lookups, but sign the full key?

Reply:
`gpg -k` ie. the default format seems to be what you expect here..

Reply:
I changed the argument from `--gpg-fpr` (fingerprint) to `--gpg-key` since we talked about not using the term fingerprint since it could bring confusion.
The idea is that the user still provides the public key fingerprint.

Reply:
Yeah, I'm referring to the "GPG long key", shouldn't we say "GPG key fingerprint" here?

Reply:
I understand that the fingerprint is the long key..
We can change it since fingerprint is a more used term for this kind of key.